src/backend/model_deployments/cohere_platform.py

```diff
@@ -29,8 +29,16 @@ def __init__(self, **kwargs: Any):
         )
         self.client = cohere.Client(api_key, client_name=self.client_name)
 
-    @property
-    def rerank_enabled(self) -> bool:
+    @classmethod
+    def name(cls) -> str:
+        return "Cohere Platform"
+
+    @classmethod
+    def env_vars(cls) -> List[str]:
+        return COHERE_ENV_VARS
```

Comment:
up to you for your previous comment - ok with putting list directly here and removing consts